packages/mdc-textfield/_mixins.scss

```diff
@@ -250,15 +262,34 @@
     z-index: 2;
   }
 
-  // stylelint-disable plugin/selector-bem-pattern
-  .mdc-text-field__input:hover ~ .mdc-text-field__idle-outline {
-    border: 1px solid;
+  .mdc-notched-outline,
+  .mdc-notched-outline__idle {
+    @include mdc-text-field-outline-corner-radius($mdc-text-field-border-radius);
   }
+}
+
+@mixin mdc-text-field-outline-color_($color) {
+  @include mdc-notched-outline-idle-color($color);
+  @include mdc-notched-outline-color($color);
```

Comment:
Add a comment that outlined version of text field wants the "idle" and "notched" version to have the same color. This covers two cases: 1) the "idle" state when a text field renders with no value 2) the "notched" state when a text field renders with an initial value